plugins/discovery-ec2/build.gradle

```diff
@@ -48,9 +48,30 @@ bundlePlugin {
   }
 }
 
+task writeJavaPolicy {
```

Comment:
Nit: Can you add "Test" or "Testing" somewhere in the task name?

Reply:
sure, addressed in b548a59